server/src/main/java/org/eclipse/lsp/cobol/core/engine/dialects/DialectUtils.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2021 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.engine.dialects;
+
+import com.google.common.collect.ImmutableMap;
+import lombok.experimental.UtilityClass;
+import org.antlr.v4.runtime.ParserRuleContext;
+import org.eclipse.lsp.cobol.core.engine.dialects.idms.IdmsDialect;
+import org.eclipse.lsp.cobol.core.messages.MessageService;
+import org.eclipse.lsp.cobol.core.model.ResultWithErrors;
+import org.eclipse.lsp.cobol.core.model.SyntaxError;
+import org.eclipse.lsp.cobol.core.model.tree.Node;
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.Range;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/** Dialect utility class */
+@UtilityClass
+public class DialectUtils {
+  private static final Map<String, CobolDialect> DIALECTS_SUPPLIERS = ImmutableMap.of(
+      IdmsDialect.NAME, new IdmsDialect()
+  );
+
+  private static final CobolDialect EMPTY_DIALECT = new EmptyDialect();
+
+  /**
+   * Process the text with dialects
```

Comment:
The parameters are not described, and the comments are not auto-formatted

Reply:
Ooops.